packages/card/index.tsx

```diff
@@ -101,6 +113,25 @@ function Heading({ as: Component = 'h2', children }: HeadingProps) {
   )
 }
 
+function ProductBadges({ badges, appearance = 'light' }: ProductBadgesProps) {
+  return (
+    <div className={s.productBadges}>
+      {badges.map((badge, stableIdx) => {
+        return (
+          // eslint-disable-next-line react/no-array-index-key
+          <React.Fragment key={stableIdx}>
```

Comment:
❓ Do we need `<React.Fragment>` here or can we add the key directly to `<ProductBadge>` ?

Reply:
If not, can we also delete the React import?

Reply:
Oops - good catch. This is cruft. [Removed](https://github.com/hashicorp/react-components/pull/748/commits/3be3b6a09666b959e6e73906a7bb10533fd0bca1)!